android/src/main/java/com/imagepicker/Options.java

```diff
@@ -10,6 +10,8 @@ public class Options {
     Boolean includeExtra;
     int videoQuality = 1;
     int quality;
+    int conversionQuality = 92;
```

Comment:
What's the reason for 92 as the defautl?

Reply:
Same as ImageMagick defaults.

https://imagemagick.org/script/command-line-options.php#quality